src/styles/_extends.scss

```diff
@@ -0,0 +1,8 @@
+%position {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
```

Comment:
Isn't it default values? Why do you need them?

Reply:
I positioned the elements in the middle of the screen in this way to make it faster... i can' t delete this values< but i can promiss that in next time I will not use this)) i will find another way

Reply:
Let's do it in this task, just use 50% with only TWO values